apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -41,11 +41,20 @@ export const PropertyAddressSearchInput: React.FC<IAddressSearchInput> = (props)
 
     const searchAddress = useCallback(
         (query, skip) => {
+            const splitted_query = query.split(/\s|\.|,/).filter(e => e != '')
```

Comment:
I think patterns (/\s|\.|,/) should be taken out as constants

Reply:
Yep, please move out static regexp in order to compile it just once and then reuse

Reply:
`/[\s.,]+/gm` this regexp would work better and you don't need `.filter(e => e != '')` part

Reply:
can you please move this to helper and add tests

Reply:
> can you please move this to helper and add tests

Tests for UX/UI?)